src/main/java/edu/harvard/iq/dataverse/settings/JvmSettings.java

```diff
@@ -91,30 +91,69 @@ public enum JvmSettings {
 
     // PERSISTENT IDENTIFIER SETTINGS
     SCOPE_PID(PREFIX, "pid"),
-    
-    // PROVIDER EZID (legacy) - these settings were formerly kept together with DataCite ones
-    SCOPE_PID_EZID(SCOPE_PID, "ezid"),
-    EZID_API_URL(SCOPE_PID_EZID, "api-url", "doi.baseurlstring"),
-    EZID_USERNAME(SCOPE_PID_EZID, "username", "doi.username"),
-    EZID_PASSWORD(SCOPE_PID_EZID, "password", "doi.password"),
+    PID_PROVIDERS(SCOPE_PID, "providers"),
+    PID_DEFAULT_PROVIDER(SCOPE_PID, "default-provider"),
+    PROVIDERS_DIRECTORY(SCOPE_PID, "provider-implementations-directory"),
+    SCOPE_PID_PROVIDER(SCOPE_PID),
+    PID_PROVIDER_TYPE(SCOPE_PID_PROVIDER, "type"),
+    PID_PROVIDER_LABEL(SCOPE_PID_PROVIDER, "label"),
+    PID_PROVIDER_AUTHORITY(SCOPE_PID_PROVIDER, "authority"),
+    PID_PROVIDER_SHOULDER(SCOPE_PID_PROVIDER, "shoulder"),
+    PID_PROVIDER_IDENTIFIER_GENERATION_STYLE(SCOPE_PID_PROVIDER, "identifier-generation-style"),
+    PID_PROVIDER_DATAFILE_PID_FORMAT(SCOPE_PID_PROVIDER, "datafile-pid-format"),
+    PID_PROVIDER_MANAGED_LIST(SCOPE_PID_PROVIDER, "managed-list"),
+    PID_PROVIDER_EXCLUDED_LIST(SCOPE_PID_PROVIDER, "excluded-list"),
+
+    // PROVIDER DATACITE (legacy - single provider setting)
+    SCOPE_LEGACY_PID_DATACITE(SCOPE_PID, "datacite"),
+    LEGACY_DATACITE_MDS_API_URL(SCOPE_LEGACY_PID_DATACITE, "mds-api-url", "doi.baseurlstring"),
+    LEGACY_DATACITE_REST_API_URL(SCOPE_LEGACY_PID_DATACITE, "rest-api-url", "doi.dataciterestapiurlstring", "doi.mdcbaseurlstring"),
+    LEGACY_DATACITE_USERNAME(SCOPE_LEGACY_PID_DATACITE, "username", "doi.username"),
+    LEGACY_DATACITE_PASSWORD(SCOPE_LEGACY_PID_DATACITE, "password", "doi.password"),
+    
+    // PROVIDER EZID (legacy - single provider setting)
+    SCOPE_LEGACY_PID_EZID(SCOPE_PID, "ezid"),
+    LEGACY_EZID_API_URL(SCOPE_LEGACY_PID_EZID, "api-url"),
+    LEGACY_EZID_USERNAME(SCOPE_LEGACY_PID_EZID, "username"),
+    LEGACY_EZID_PASSWORD(SCOPE_LEGACY_PID_EZID, "password"),
+    
+    // PROVIDER PERMALINK (legacy - single provider setting)
+    SCOPE_LEGACY_PID_PERMALINK(SCOPE_PID, "permalink"),
+    LEGACY_PERMALINK_BASEURL(SCOPE_LEGACY_PID_PERMALINK, "base-url", "perma.baseurlstring"),
+
+    // PROVIDER HANDLE (legacy - single provider setting)
+    SCOPE_LEGACY_PID_HANDLENET(SCOPE_PID, "handlenet"),
+    LEGACY_HANDLENET_INDEX(SCOPE_LEGACY_PID_HANDLENET, "index", "dataverse.handlenet.index"),
+    SCOPE_LEGACY_PID_HANDLENET_KEY(SCOPE_LEGACY_PID_HANDLENET, "key"),
+    LEGACY_HANDLENET_KEY_PATH(SCOPE_LEGACY_PID_HANDLENET_KEY, "path", "dataverse.handlenet.admcredfile"),
+    LEGACY_HANDLENET_KEY_PASSPHRASE(SCOPE_LEGACY_PID_HANDLENET_KEY, "passphrase", "dataverse.handlenet.admprivphrase"),
+    
+    // PROVIDER EZID - these settings were formerly kept together with DataCite ones
+    SCOPE_PID_EZID(SCOPE_PID_PROVIDER, "ezid"),
```

Comment:
For this and the other providers: I'm not yet sure I like this sub-scoping. It wasn't necessary for `dataverse.files.*`, maybe it isn't necessary here, too? It would be more aligned if not, lot less typing, etc. How large is the risk of collisions?

That said: the `dataverse.files.*` MPC-enablement is still a TODO. If you feel we should do more sub-scoping for these, too, this is fine!

Reply:
I'm not set on it, but to me the benefit is not in avoiding collisions, it's in making it clear which entries you need for a given type of provider. EZID doesn't have mds and rest api URLs, Permas don't have username/password, nothing but handles get a key, etc. I think we have the same issue with stores - direct up/download only apply to S3 stores, etc.

Reply:
BTW - I think you gave me the idea with https://github.com/IQSS/dataverse/blob/4b5cbe865d215e4f4c8dcf58c28b5cf3af98d414/src/main/java/edu/harvard/iq/dataverse/settings/JvmSettings.java#L115